lib/Entity/DataSet.php

```diff
@@ -514,6 +515,11 @@ public function getData($filterBy = [], $options = [])
                             // Pull out the column name and date format
                             $details = explode(',', str_replace(')', '', str_replace('$dateFormat(', '', $column->formula)));
 
+                            if (isset($details[2])) {
+                                $language = str_replace(' ', '', $details[2]);
+                            }
+
+                            $this->date->setLocale($language);
```

Comment:
does this set the locale to `""` if the language isn't specified?

Reply:
If it's not set in Formula, then yes, however in the end it would be set to en - as that's the default for the date translator.

Perhaps we should do something like that (covers both your comments) 🤔 

``` 
                            if (isset($details[2])) {
                                $language = str_replace(' ', '', $details[2]);
                            } else {
                                $language = $this->config->getSetting('DEFAULT_LANGUAGE', 'en_GB');
                            }
```
This seems to correctly translate to the language set in Formula and if it is not set there, then it translates to the CMS language.

Reply:
that sounds good!